components/Footer/index.js

```diff
@@ -43,8 +43,27 @@ class Footer extends React.Component {
 
     const getYear = new Date().getFullYear();
 
+    if (this.context.router.getCurrentPathname() === '/contact'){
```

Comment:
@KadoBOT you don't need this extra conditional and duplicated footer. Your other conditional already returns the correct footer on the contact page.